vxyowsup/whatsapp.py

```diff
@@ -60,11 +60,11 @@ def setup_transport(self):
 
     @defer.inlineCallbacks
     def teardown_transport(self):
-        print "Stopping client ..."
+        log.debug("Stopping client ...")
```

Comment:
I've changed all the `debug` logs to `info` in `teardown_transport`, `catch_exit` and `client_stop`. There's a `log.info` in `handle_outbound_message` which was there in the skeleton, but it seems to fit in better with what we've got in the debug logs..?